pkg/target/manager.go

```diff
@@ -30,9 +30,13 @@ type Manager interface {
 	CurrentTarget() (Target, error)
 
 	TargetGarden(name string) error
+	DropTargetGarden() (string, error)
```

Comment:
could you add a go doc for all the exported functions / constants? at least for the newly introduced ones..